### PR TITLE
feat: add prev page button component

### PR DESCRIPTION
**Pull request by LenaYork**
Add Prev Step Button component 
image

### CODE REVIEW

File: src/components/ui/PrevStepButton.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { Box, Button } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+type PreviousStepButtonProps = {
+  onSubmit: (action: string) => void
+}
+
+const PrevStepButton = (props: PreviousStepButtonProps) => {
+  const { onSubmit } = props
```
Comment (LyubovSugak) [inline src/components/ui/PrevStepButton.tsx:9]: maybe makes sense to change the naming, bc it's not submit button - onHadleStepBack or smth similar

Comment (LenaYork) [reply src/components/ui/PrevStepButton.tsx]: Agree. Changed into stepBackHandler

File: .env
Comment (LyubovSugak) [inline .env:1]: add .env file to gitignore. On master it's located there already.

**Review by LyubovSugak — CHANGES_REQUESTED** [review]

File: .env
Comment (LenaYork) [reply .env:1]: - [x] done

**Pull request closed without merging**